bbj-vscode/src/language/bbj-completion-provider.ts

```diff
@@ -51,11 +50,11 @@ export class BBjCompletionProvider extends DefaultCompletionProvider {
     /**
      * Need to override to control deduplicate filtering
      */
-    protected override completionForCrossReference(context: CompletionContext, crossRef: NextFeature<CrossReference>, acceptor: CompletionAcceptor): MaybePromise<void> {
-        const assignment = getContainerOfType(crossRef.feature, isAssignment);
+    protected override completionForCrossReference(context: CompletionContext, crossRef: NextFeature<GrammarAST.CrossReference>, acceptor: CompletionAcceptor): MaybePromise<void> {
+        const assignment = AstUtils.getContainerOfType(crossRef.feature, GrammarAST.isAssignment);
         let node = context.node;
         if (assignment && node) {
-            if (crossRef.type && (crossRef.new || node.$type !== crossRef.type)) {
+            if (crossRef.type) {
```

Comment:
is `(crossRef.new || node.$type !== crossRef.type)`  obsolet now?

Reply:
I looked it up in the Langium equivalent, there the same change was made.

Reply:
> When `type` is set, it indicates that we have just entered a new parser rule.

Well, okay.